config/routes.rb

```diff
@@ -121,6 +123,7 @@
   match 'resources', to: 'pages#resources', via: :get
   get 'home_data', to: 'pages#home_data', defaults: { format: 'json' }
 
+
```

Comment:
👾

Reply:
I guess I just have to remove the line, right?